src/styles/interactives.scss

```diff
@@ -0,0 +1,96 @@
+$raise-correct-answer-border-color: #019920;
+$raise-selected-focus-border-color: #159eb2;
+$raise-wrong-answer-border-color: #fc0516;
+$raise-default-answer-border-color: #424242;
+$raise-medium-gray: #626262;
+$raise-light-gray: #d9d9d9;
+$raise-answer-background: #f5f5f5;
+
+div.os-raise-grid {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-gap: 2rem;
+}
+
+div.os-raise-default-answer-choice {
+  border: 2px solid $raise-default-answer-border-color;
+  border-radius: 5px;
+  box-shadow: 0 2px 2px #808080;
+}
+
+div.os-raise-default-answer-choice:focus-within {
+  border-color: $raise-selected-focus-border-color;
+}
+
+div.os-raise-default-answer-choice > div:hover {
+  background-color: $raise-answer-background;
+}
+
+div.os-raise-no-box-shadow {
+  box-shadow: none;
+}
+
+div.os-raise-selected-answer-choice {
+  border-color: $raise-selected-focus-border-color;
+  background-color: $raise-answer-background;
+}
+
+div.os-raise-correct-answer-choice {
+  border-color: $raise-correct-answer-border-color;
+}
+
+div.os-raise-wrong-answer-choice {
+  border-color: $raise-wrong-answer-border-color;
+}
+
+label.os-raise-fill-label-container {
+  width: 100%;
+  height: 100%;
+  padding: 1rem 0.5rem 1rem 0.8rem;
+  cursor: pointer;
+}
+
+input.os-raise-hide-input-button {
+  clip: rect(0 0 0 0);
+  clip-path: inset(100%);
+  height: 1px;
+  overflow: hidden;
+  position: absolute;
+  white-space: nowrap;
+  width: 1px;
+}
+
+div.os-raise-feedback-message {
+  background-color: $raise-light-gray;
+  padding: 0.5rem;
+  border-radius: 5px;
+}
+
+div.os-raise-feedback-message p {
+  color: $raise-medium-gray;
+}
+
+p.os-raise-attempts-text {
+  color: $raise-medium-gray;
+}
+
+div.os-raise-justify-content-end {
+  justify-content: end;
+}
+
+.disabled {
```

Comment:
This is just an example, but I feel like we have some inconsistency in our class naming conventions currently. Here is what (I think) we were doing before:

* Classes that are "public" (e.g. documented for authors and defined in `content.scss`): Prefix with `os-raise-`
* Classes that were "internal" (previously there were just [these two](https://github.com/openstax/k12-apps-raise/blob/4c9ca86e9adab9272288c6b90d98582d0e819d5b/src/styles/bootstrap.scss#L95-L107), so it's arguable whether that constitutes a "convention" or just inconsistency): Prefix with `os-`
    * I guess `.os-raise-bootstrap` kind of violates this "convention" :thinking: 

In any case, thoughts on what we want to try and do moving forward @P-Gill97 , @MReyna12 ? Maybe everything should be `os-raise-*`? Or do we want to differentiate "internal" vs "external" somehow?

Reply:
Regarding this specific example - I can revise to `os.raise-disabled`. 

I believe you and/or Prabhdip may have mentioned the public/internal concept, but I may have misunderstood the prefix `os-raise` versus `os-`. 

So public classes are limited to classes in `content.scss`. This would not include styles in `interactives.scss`, correct? Since these classes would be only for engineer use and not content authors. If this is the case and we want to differentiate between internal / external, then I am okay with the `os-` prefix for internal classes and `os-raise-`.